src/quickgui/qgsquickidentifykit.h

```diff
@@ -66,6 +66,13 @@ class QUICK_EXPORT QgsQuickIdentifyKit : public QObject
      */
     Q_PROPERTY( int featuresLimit READ featuresLimit WRITE setFeaturesLimit NOTIFY featuresLimitChanged )
 
+    /**
+     * Indicator if TopDownStopAtFirst mode for QgsQuickIdentifyKit::identify() is on
```

Comment:
documentation needs to be improved. I would be completely confused without reading the c++ file. Also documentation must be added to identify() as for other parameters. 

 Also why it is needed ? what is the use case for this behaviour? Cannot be identifyOne used without layer parameter for the use case?

Reply:
The parameter and related functionality fixes: "Identification should give preference to point features. Right now it is difficult or impossible to identify a pot - most of the time I get a drop trench or pot-property link instead."

IdentifyOne is currently used always without layer parameter. Therefore identifytool search among all layers and returns all "selected" features which are passed to _closestFeature function. Since there are features from all point and polyline layers, in most cases is polyline feature selected as the closest one from screen point (obviously).
When TopDownStopAtFirst parameter is used, is searches from top layers which are point layers. It stops identifying on first bunch of features search "TopDownStopAtFirst" layer. Indeed, a feature from first layer will be always selected in a high feature density project which is not ideal neither.

However, at least it fixes reported issue.